dev/com.ibm.ws.app.manager.war/resources/com/ibm/ws/app/manager/war/internal/resources/Messages.nlsprops

```diff
@@ -220,3 +220,20 @@ warning.could.not.expand.application.useraction=Attempt to resolve the issue in
 error.file.outside.archive=CWWKZ0132W: The file {0} in the archive {1} will not be extracted because the path is not contained within the archive.
 error.file.outside.archive.explanation=The file's path specifies a relative directory. Extracting the file would result in the file being located outside of the archive's root directory.
 error.file.outside.archive.useraction=Remove any files with relative paths from the archive.  
+
+#{0} is the application name, {1} is the location of the EAR or WAR file, {2} is the location where the file is expanded
+info.expanding.app=CWWKZ0133I: The application named {0} at location {1} is being expanded to the directory {2}.
+info.expanding.app.explanation=The archive file for the application is being expanded because automatic expansion of applications was configured.
+info.expanding.app.useraction=No action is required.
```

Comment:
~~No action is required.~~
For informational messages, leave the useraction filed blank.